src/main/kotlin/com/atlassian/performance/tools/virtualusers/api/VirtualUserActivity.kt

```diff
@@ -0,0 +1,21 @@
+package com.atlassian.performance.tools.virtualusers.api
+
+import com.atlassian.performance.tools.jiraactions.api.ActionMetric
+import com.atlassian.performance.tools.jiraactions.api.ActionType
+
+/**
+ * @since 3.11.0
+ */
+class VirtualUserActivity internal constructor(
+    val type: ActivityType,
+    val metric: ActionMetric
+)
+
+enum class ActivityType(
```

Comment:
Just a reminder that usage of enum will limit future evolution:
https://github.com/wyrzyk/java-api-design-checklist/blob/master/CHECKLIST.md#255-do-not-add-new-values-to-a-released-enumeration

Reply:
Initially I thought that I exhausted all possible VU activities and therefore I won't have to add new `ActivityType`s.
But that is just a failure of my imagination. For example, we could conceivably add `USER_GENERATION` activity or any other future feature like session switching.
- [ ] avoid enum